src/Utils/llm_utils.py

```diff
@@ -10,6 +10,10 @@
 from langchain_openai.chat_models.base import ChatOpenAI
 from pydantic import BaseModel
 
+# from langchain.globals import set_debug
+
+# set_debug(True)  # Enables LangChain debug mode globally
```

Comment:
When should anyone uncomment these? If there is no practical use case, let's remove these lines